step-release-vis/src/app/components/tooltip/tooltip.ts

```diff
@@ -11,6 +11,10 @@ import {CandidateService} from '../../services/candidateService';
 export class TooltipComponent implements OnInit {
   @Input() tooltip: Tooltip = new Tooltip();
   currentSnapshot: Snapshot;
+  width = 200;
+  height = 50;
+
+  // TODO(#234): Compute width/height of the tooltip according to the data.
```

Comment:
comment related to a code segment should come before the code, unless they are all in the same line